.babelrc

```diff
@@ -8,15 +9,16 @@
     "transform-class-properties",
     "transform-export-extensions",
     "lodash",
-    "transform-runtime"
   ],
   "presets": [
     ["env", {
       "targets": {
         "browsers": [
-          "last 2 versions",
-          "safari >= 7"
-        ]
+          ">0.25%",
+          "not dead",
+          "ie >= 11" // Office 365 support
+        ],
+        "node": "6"
```

Comment:
just 6 builds compatibility for the latest version of 6 (so 6.17) when this should be either 6.0 or 8 (because it's gonna the be equivalent of 6 in a couple of months)